motor/discovery/k8s/resolver.go

```diff
@@ -351,6 +353,17 @@ func addSeparateAssets(
 		resolved = append(resolved, admissionReviews...)
 	}
 
+	// discover ingresses
+	if tc.IncludesOneOfDiscoveryTarget(common.DiscoveryAll, DiscoveryIngresses) {
```

Comment:
Is it intentional that ingresses are not included in the `auto` discovery? That is the default discovered assets

Reply:
Yes. In order to avoid the assets being unscored, it is not part of auto for now. When we have policies written to execute against Ingress resources, then we can make Ingress part of "auto".

Reply:
@imilchev My hope is that next week we'd get at least one policy written so we could turn this on by default.